src/react-components/room/ChatSidebarContainer.js

```diff
@@ -102,18 +113,23 @@ function updateMessageGroups(messageGroups, newMessage) {
 export function ChatContextProvider({ messageDispatch, children }) {
   const [messageGroups, setMessageGroups] = useState([]);
   const [unreadMessages, setUnreadMessages] = useState(false);
+  const isMod = useCan("kick_users");
 
   useEffect(
     () => {
       function onReceiveMessage(event) {
         const newMessage = event.detail;
+
+        if (isMod && newMessage.type === "permission") return;
```

Comment:
Wouldn't we still want to see this in the log? Especially if a different moderator toggled it.

Reply:
I don't think we really considered that case. I've updated so other moderators also can see the permission updates notifications.